docs/src/modules/sandbox/StackBlitz.ts

```diff
@@ -1,99 +1,202 @@
-import addHiddenInput from 'docs/src/modules/utils/addHiddenInput';
-import { CODE_VARIANTS } from 'docs/src/modules/constants';
+import sdk from '@stackblitz/sdk';
```

Comment:
Is this worth 3KB gzipped? https://bundlephobia.com/package/@stackblitz/sdk@1.11.0 What complexity does it solve that we really need this for? I mean what's wrong with making a call to the HTTP API ourselves directly?

Reply:
So the source looks simple https://github.com/stackblitz/sdk/blob/614e61fe8f50c419ed8414024899e37e69b1f999/src/generate.ts#L89. Maybe it does tree-shake correctly?

https://bundlejs.com/?q=%40stackblitz%2Fsdk%401.11.0&treeshake=%5B*%5D&text=%22export+default+sdk.openProject%3B%22

Reply:
I removed the sdk again. We're downloading 3MB of resources on initial load so optimizing for 2.5kB is not really going to be noticable 😬 but it's the principle.

Reply:
> We're downloading 3MB of resources on initial load

Gzipped? Oh wow, probably part of why the docs is so slow 🙈